repose-aggregator/tests/functional-tests/src/integrationTest/groovy/features/filters/apivalidator/ApiValidatorMultiTenantTest.groovy

```diff
@@ -124,9 +124,9 @@ class ApiValidatorMultiTenantTest extends ReposeValveTest {
             url: "$reposeEndpoint/v1/resource/other",
             method: "GET",
             headers: [
-                "X-Tenant"   : "1,5",
-                "X-Other"    : "other",
-                "X-Map-Roles": mapHeaderValue])
+                (TENANT_ID)       : "1,5",
+                (HDR_OTHER)       : VAL_OTHER,
+                (TENANT_ROLES_MAP): mapHeaderValue])
```

Comment:
Just to show the difference here, you could do something like:
```
headers: ([(TENANT_ID): "1,5"] + standardHeaders - [(ROLES): "foo,bar"]))
```

Although , I'm kind of annoyed that I can't find a way to remove the `ROLES` header by key rather than by entry. Well, there's `remove`, but that mutates the map and returns the removed value, which means it can't be done in a single expression.